src/components/graph/search/GraphSearch.tsx

```diff
@@ -83,7 +83,7 @@ function GraphSearch(props: Props) {
 					inputConfig.isValidDate &&
 					inputConfig.timestamp <= moment.utc().valueOf()
 				) {
-					onTimestampSubmit(inputConfig.timestamp);
+					onTimestampSubmit(inputConfig.timestamp, true);
```

Comment:
well with the same success we could have deleted this if block with function because now it never gets called, right?. But I'm confused about why did we even had this and what is expected behavior